single.php

```diff
@@ -34,13 +34,13 @@
 									endif;
 
 									if(get_the_content()):
-										echo "<p>".get_the_content()."</p>";
+											the_content();
```

Comment:
Good question. Actually, it should. However, i just a bit concern about the blank post the contains only the P tag. The the_content() and get_the_content(), their function seems like, but sometimes, they provide the different result as Caption case. Let me test it first.